### PR TITLE
New Pull for adding a windows 11 clone 

### PR DESCRIPTION
**Pull request by ScorchingShade**

### CODE REVIEW

**Comment by ScorchingShade** [comment]
bitmoji

So please merge!

File: projects/ScorchingShade/win11exp.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+name: Windows 11 Demo
+repoUrl: https://github.com/ScorchingShade/Windows11Clone
+description: A windows 11 clone homepage.
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - HTML / CSS
+  - nextjs
```
Comment (ramonh) [inline projects/ScorchingShade/win11exp.mdx:8]:
```suggestion
  - NextJS
```

Capitalising, just for consistency's sake :)

File: projects/ScorchingShade/win11exp.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  - Developers
+  - Frontend Developers
+  - Designers
+  - Maintainers
+  - Code Reviewers
+  - Technical Writers
+  - Mentors
+  - Testers
+featuredMap: # (optional) A CodeSee Map to serve as a visual starting point for contributors
+  url: https://app.codesee.io/maps/public/956b3340-22ad-11ec-bd98-4f6e10139d3f
```
Comment (ramonh) [inline projects/ScorchingShade/win11exp.mdx:28]: Please see my above comment about setting the Map to public! I can't seem to open this one

**Review by ramonh — CHANGES_REQUESTED** [review]
Looking good, thanks @ScorchingShade! Could you please take a look at my feedback below?

Thanks in advance!

File: projects/ScorchingShade/win11exp.mdx (hunk 2 above)
Comment (ScorchingShade) [reply projects/ScorchingShade/win11exp.mdx:28]: Hey @ramonh ,This should work now, I changed the privacy settings for both urls

File: projects/ScorchingShade/win11exp.mdx (hunk 1 above)
Comment (ScorchingShade) [reply projects/ScorchingShade/win11exp.mdx:8]: Thank you! I was so confused about the nomenclature here!

File: projects/ScorchingShade/nextEvents.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  - Frontend Developers
+  - Designers
+  - Maintainers
+  - Code Reviewers
+  - Technical Writers
+  - Backend Developers
+  - Mentors
+  - Testers
+featuredMap: # (optional) A CodeSee Map to serve as a visual starting point for contributors
+  url: https://app.codesee.io/maps/f391b0d0-22b1-11ec-8415-678e910efb84
```
[the remark this answers is not in the document]
Comment (ScorchingShade) [reply projects/ScorchingShade/nextEvents.mdx:32]: Hey @ramonh this is updated, can you please verify if I need to push again?

**Review by ScorchingShade — COMMENTED** [review]
This should work now

**Comment by ramonh** [comment]
Most excellent, thanks @ScorchingShade!

Welcome to Port! 🚢

**Review by ramonh — APPROVED** [review]

**Pull request merged**